### PR TITLE
Adds login instructions

### PR DESCRIPTION
**Pull request by aichamorro**
# Introduction
This PR adds the instructions the login exercise. It adds an explanation on the UI/UX of the component, plus some videos to show what needs to be implemented.

This PR also adds some styling to the images, so they appear centered. Besides, it disables the text fields to avoid distractions during the workshop (the keyboard popping up)

### CODE REVIEW

File: docs/Exercises.md [hunk 1]
```diff
@@ -17,4 +17,34 @@ To implement it, open the `AnimatableRowView/index.tsx` file and look for the TO
...
+We will be editing the following files:
+ * `storybook/Login/LoginScreen/index.tsx`: This has the code for the component.
+ * `storybook/Login/LoginScreen/style.ts`: This file has the styling rules for the `LoginScreen` component.
+
+We suggest the following order to implement the exercise:
+1. Hide the login form
+2. Move the logo to the center of the screen.
+3. Add an animation to move the logo from the center to its final position
+4. Add an animation to show the login form
+5. Add an animation when toggling the error (by pressing the Login button)
```
Comment (juanjo-ramos) [inline docs/Exercises.md]: Is it mentioned somewhere (code maybe) that they should use `LayoutAnimation` for this? If not, I think we should.

Comment (aichamorro) [reply docs/Exercises.md]: there is the following comment in the code
```
    // TODO:
    // 4. Animate the transition of the error message
```

File: docs/Exercises.md [hunk 2]
```diff
@@ -17,4 +17,34 @@ To implement it, open the `AnimatableRowView/index.tsx` file and look for the TO
 If you get stuck just ask. You also have the full solution in the `FINAL-Exercise-One` branch. 
 
 ## Exercise two: Animation on the login screen
-TBC
+### Introduction
+In this exercise, we are re-creating [Hudl's Login screen (iOS version)](https://drive.google.com/file/d/19tI6PLJ_ZQCnzGYfdmcsljKlbua3MzW9/view?usp=sharing):
+
+<div style="text-align: center"><img src="https://user-images.githubusercontent.com/6965651/55230257-8af0a000-521f-11e9-8090-2c1b17ef76ad.gif" /></div>
+
+As you can see, the logo stays in the middle of the screen while the app loads. Then, it moves to the top and the login form fades in. Even more, when [there is an error](![Hudl-Login](https://user-images.githubusercontent.com/6965651/55230257-8af0a000-521f-11e9-8090-2c1b17ef76ad.gif)):
```
[the remark this answers is not in the document]
Comment (aichamorro) [reply docs/Exercises.md]: it should not end there...

File: docs/Exercises.md (hunk 1 above)
Comment (aichamorro) [reply docs/Exercises.md]: I left hint too:
```
   1. Hint: we saw there is an easy way to animate layout changes when adding/removing elements...
```

**Comment by aichamorro** [comment]
I can add the cheatsheet in a different PR

**Pull request merged**